python/graphstorm/trainer/lp_trainer.py

```diff
@@ -34,16 +34,44 @@
 from ..utils import barrier, is_distributed
 
 class GSgnnLinkPredictionTrainer(GSgnnTrainer):
-    """ Link prediction trainer.
+    """ A trainer for link prediction
 
-    This is a highlevel trainer wrapper that can be used directly to train a link prediction model.
+    This is a high-level trainer wrapper that can be used
+    directly to train a link prediction model.
+
+    It makes use of the functions provided by `GSgnnTrainer`
+    to define two main functions: `fit` that performs the training
+    for the model that is provided when the object is created,
+    and `eval` that evaluates a provided model against test and
+    validation data.
 
     Parameters
     ----------
-    model : GSgnnLinkPredictionModelBase
+    model : GSgnnLinkPredictionModel
```

Comment:
Could you double check the other trainers' input model, and change from **ModelBase to **Model?

Reply:
Yup, fixed in ep_trainer with last commit.